ui/app/controllers/exec.js

```diff
@@ -20,9 +21,17 @@ export default Controller.extend({
   socketOpen: false,
   taskState: null,
 
-  runningAllocations: filterBy('model.allocations', 'isRunning'),
-  runningTaskGroups: mapBy('runningAllocations', 'taskGroup'),
-  uniqueRunningTaskGroups: uniq('runningTaskGroups'),
+  pendingAndRunningAllocations: computed('model.allocations.@each.clientStatus', function() {
+    return this.model.allocations.filter(
+      allocation => allocation.clientStatus == 'pending' || allocation.clientStatus == 'running'
```

Comment:
loose equality? 😱

Reply:
hmm indeed, I’m surprised there wasn’t a lint error on that. Fixed, thanks. I’m still trying to fix the bug I mentioned in my latest comment, though, sadly.